SignalServiceKit/src/Messages/Attachments/TSAttachmentStream.m

```diff
@@ -703,6 +703,75 @@ - (nullable TSAttachmentStream *)cloneAsThumbnail
     return thumbnailAttachment;
 }
 
+// MARK: Protobuf serialization
+
++ (nullable OWSSignalServiceProtosAttachmentPointer *)buildProtoForAttachmentId:(nullable NSString *)attachmentId
+                                                                 isVoiceMessage:(BOOL)isVoiceMessage
+{
+    OWSAssert(attachmentId.length > 0);
+
+    // TODO we should hoist this transaction to make sure we're getting a consistent view in the message sending process
```

Comment:
What does `hoist` mean here?  Pass in/plumb through?

Reply:
Correct - I'll update the comment.